vendor/github.com/status-im/ottoext/loop/loop.go

```diff
@@ -5,7 +5,7 @@ import (
 	"sync"
 	"sync/atomic"
 
-	"github.com/robertkrimen/otto"
+	"github.com/status-im/status-go/geth/jail/vm"
```

Comment:
Hm, actually I haven't checked these files in `vendor`. 

I would try to avoid this kind of cross-package dependencies. Why do we need `jail/vm.Vm` here instead of `otto.Otto`? I though we are leaving this package not thread-safe and implement thread-safety only in `status-go`.

Reply:
@adambabik the answer is a bit long for the comment, but I think I outlined the problem in this issue: https://github.com/status-im/status-go/issues/289. Do you need more context?

Reply:
Yes, it's clear. However, if we need thread-safety in this package, shouldn't we implement it here?

Reply:
The thing is there are other packages that indirectly utilize vm's unsafe Get/Set/Run/Calls - `timers` and `fetch` namely and they also need to be guarded for safe execution, hence this wrapper.

Reply:
@divan is right on that point, previously we had use `JailCell/Cell` has the means to guard all access to vm, am very much liking the idea of not having to expose `JailCell/Cell` just because of the need to use otto.